### ISSUE TITLE
Toggle estimation settings to turn off estimation on PR's

### ISSUE THREAD

**Issue by benschac** [ZenHubIO/support#432]
I would like the option of turning off estimates on PR's.  Our team strictly estimates tickets to measure business value effort of each task.

**Comment by aupright** [comment]
Thanks for the feedback @benschac - is this just a visual preference of not seeing the estimate selector inside of the PR view? It sounds like your team has already adopted the practice of not using the 'estimates' with regards to PRs :smiley:

**Comment by benschac** [comment]
Yes, just not showing that estimation selector.  Yeah, but it's still can cause confusion.  Also, business value shouldn't be assigned to code, it should be assigned to an issue with business value. :)

**Comment by aupright** [comment]
Got it, thanks for the additional context here @benschac. Let's see if we can get some feedback from the rest of the community on whether other users would find this useful as well!

**Comment by driesvints** [comment]
Just stumbled upon this. I also think it makes more sense to disable estimates all together on an organization if they don't use them. We use Kanban and the current story point estimates really have no value to us.

What would be even better would estimates could be tracked in different ways: due dates, etc and with better and extended chart support (see various other issues in this repo).
